test/integration/patients/list/patients-all.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'underscore';
+
+context('patient sidebar', function() {
+  specify('display patient data', function() {
+    cy
+      .server()
+      .routePatient((fx) => {
+        fx.data = {
+          id: 1,
+          attributes: {
+            first_name: 'First',
+            last_name: 'Last',
+          },
+        };
+
+        return fx;
+      })
+      .routePatients((fx) => {
+        _.each(fx.data, (patient, index) => {
```

Comment:
I think you can get away with either `unshift`ing a patient into the data to test, or just modify the first one.  The rest of the data can be random